include/Smarty/plugins/function.sugar_include.php

```diff
@@ -78,14 +80,26 @@ function smarty_function_sugar_include($params, &$smarty)
 	   	  foreach($params['include'] as $include) {
 	   	  	      if(isset($include['file'])) {
 	   	  	         $file = $include['file'];
```

Comment:
I think if you add something like this is more handy to see if something went wrong at this point:
```
$realpath = realpath($include['file']);
if (!$realpath) {
  LoggerManager::getLogger()->error('smarty_function_sugar_include - Given file path is incorrect: ' . $include['file']);
} else {
  $file = $realpath;
  ...
```

Reply:
Done